tests/e2e-tests/README.md

```diff
@@ -6,108 +6,180 @@ Automated end-to-end tests for WooCommerce.
 
 ```
-const WP_ADMIN_USER = {
- 	username: 'admin',
- 	password: 'password',
- };
- 
- const {
- 	WP_USERNAME = WP_ADMIN_USER.username,
- 	WP_PASSWORD = WP_ADMIN_USER.password,
- 	WP_BASE_URL = 'http://localhost:8889',
- } = process.env;
- 
- export {
- 	WP_ADMIN_USER,
- 	WP_USERNAME,
- 	WP_PASSWORD,
- 	WP_BASE_URL,
- };
-```    
+{
+  "url": "http://localhost:8084/",
+  "users": {
+    "admin": {
+      "username": "admin",
+      "password": "password"
+    },
+    "customer": {
+      "username": "customer",
+      "password": "password"
+    }
+  }
+}
+```
 
-As per above, create an Admin user on the site and set its username and password:
+If you changed either site URL or one of the users details in the  `docker-compose.yaml` file, you'd need to copy the content of the `default.json`, paste it to `test:e2e.json` and edit it further there to match your own setup. 
 
-- username: `admin`
-- password: `password`
+## Jest test sequencer
 
-Specify base URL and Test user details using environment variables. 
+[Jest](https://jestjs.io/) is being used to run e2e tests. By default, jest runs tests ordered by the time it takes to run the test (the test that takes longer to run will be run first, the test that takes less time to run will run last). Jest sequencer introduces tools that can be used to specify the order in which the tests are being run. In our case, they are being run in alphabetical order of the directories where tests are located. This way, tests in the new directory `activate-and-setup` will run first. 
 
-#### Environment variables
+Setup Wizard e2e test (located in `activate-and-setup` directory) will run before all other tests. This will allow making sure that WooCommerce is activated on the site and for the setup wizard to be completed on a brand new install of WooCommerce. 
 
-Set environmental variables as shown below. Note that you don't need to add the trailing slash ('/') at the end of the site URL:
+## Running tests
 
-- `export WP_BASE_URL={your site URL}`
-- `export WP_USERNAME={your Test user username}`
-- `export WP_PASSWORD={your Test user password}`
+### Prep work for running tests
 
-You can unset the variables when you are done:
+- Checkout the branch to test and stay on this branch. 
 
-- `unset WP_BASE_URL`
-- `unset WP_USERNAME`
-- `unset WP_PASSWORD`
+- Run `nmp install`
 
-## Running tests
+- Run `composer install --no-dev`
 
-### How to run tests
+- Run `npm run build`
 
-To run e2e tests use the following command:
+- Run the following command to build the test site using Docker: `docker-compose up` and watch the site being built. Note that it may take a few minutes the first time you do that. The process is considered completed when the messages letting you know that WordPress was installed, WooCommerce was activated and users created will be displayed:
```

Comment:
As far as I understand, when you run `docker-compose up` for the first time, it includes `--build` by default. It is only if you need to rebuild existing container, you'd include `--build`. Hence, I think we can leave it as-is.

Reply:
I see. I was thinking more from the perspective of forgetful myself coming back after some time of not running this regularly, looking through this guide. I thought I might forget to run with the `--build` switch. In other words, running using that switch all the time seems safer to me, if I want to avoid having strange problems, so I thought why not add it here as well.